app/src/utils/Fips.ts

```diff
@@ -64,3 +64,51 @@ export const STATE_FIPS_MAP: Record<string, string> = {
     "72": "Puerto Rico",
     "78": "Virgin Islands"
 */
+
+export type Severity = "INFO" | "WARNING" | "ERROR";
+
+class Fips {
+  code: string;
+  countyName: string;
+
+  constructor(code: string, countyName: string = "") {
+    this.code = code;
```

Comment:
nit: would be nice to do some validation. You can use the regex: `^(\d{2}|\d{5})$`